api/envoy/api/v2/rds.proto

```diff
@@ -44,7 +45,18 @@ service RouteDiscoveryService {
   }
 }
 
-// [#comment:next free field: 9]
+// * HTTP :ref:`router filter <config_http_filters_router>`
+// The resource_names_subscribe field in DeltaDiscoveryRequest contains a list of virtual host names or aliases
```

Comment:
`*resource_names_subscribe*` (or do a `:ref:` link to the field name).

Reply:
fixed